src/components/lib/posix/posix.c

```diff
@@ -263,23 +272,26 @@ cos_set_tid_address(int *tidptr)
  * };
  */
 
-void* backing_data[SL_MAX_NUM_THDS];
+void* backing_data[MAX_NUM_THREADS];
 
 static void
-setup_thread_area(struct sl_thd *thread, void* data)
+setup_thread_area(void *thread, void* data)
 {
+#if 0
 	struct cos_compinfo *ci = cos_compinfo_get(cos_defcompinfo_curr_get());
 	thdid_t thdid = sl_thd_thdid(thread);
 
 	backing_data[thdid] = data;
 
 	cos_thd_mod(ci, sl_thd_thdcap(thread), &backing_data[thdid]);
+#endif
+	return ;
 }
 
 int
 cos_set_thread_area(void* data)
 {
-	setup_thread_area(sl_thd_curr(), data);
+	setup_thread_area(NULL, data);
```

Comment:
Would this work if we pass NULL into this function?

Reply:
This commit is just for removing `sl` in libposix. This of course will not work. Also, this function actually is not used anywhere. Thus passing NULL is just for pass the compiler.